src/main/java/net/dv8tion/jda/api/events/guild/scheduledevent/update/GuildScheduledEventUpdateDescriptionEvent.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2015 Austin Keener, Michael Ritter, Florian Spieß, and the JDA contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.dv8tion.jda.api.events.guild.scheduledevent.update;
+
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.GuildScheduledEvent;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+/**
+ * Indicates the {@link GuildScheduledEvent#getDescription() description} of a {@link GuildScheduledEvent} has changed.
+ *
+ * <p>Can be used to detect when the {@link GuildScheduledEvent} description has changed and retrieve the old one.
```

Comment:
I'ld rephrase this, because `retrieving` kind of implies making a RestAction getting data in JDA.
If JDA itself uses this wording at other places, then it might be fine, but i personally would prefer `viewing the old cached information`.